plugins/apitracing/src/lib/ApiTracing.cpp

```diff
@@ -0,0 +1,121 @@
+            false,
+            "",
+            "/path/to/functionDefinitions.yaml",
+            cmd};
+        cmd.parse(args);
+
+        pluginInterface->logMessage(VmiCore::Plugin::LogLevel::debug,
+                                    LOG_FILENAME,
+                                    "ApiTracing plugin version info: " + std::string(PLUGIN_VERSION) + "-" +
+                                        std::string(BUILD_VERSION));
+
+        auto tracingTargetsParser = std::make_shared<TracingTargetsParser>();
+        try
+        {
+            std::shared_ptr<IConfig> configuration = std::make_shared<Config>(config->configFilePath().value());
+            auto tracingInformation = tracingTargetsParser->getTracingTargets(configuration->getTracingTargetsPath());
+            auto functionDefinitions = std::make_shared<FunctionDefinitions>(functionDefinitionsPath);
+            functionDefinitions->init();
+            switch (pluginInterface->getIntrospectionAPI()->getOsType())
+            {
+                case VmiCore::OperatingSystem::WINDOWS:
+                {
+                    auto library = std::make_shared<Windows::Library>();
+                    tracer = std::make_shared<Tracer>(
+                        pluginInterface, configuration, tracingInformation, functionDefinitions, library);
+                    break;
+                }
+                default:
+                {
+                    throw std::runtime_error("Unknown operating system.");
+                }
+            }
+        }
+        catch (const ConfigException& exc)
+        {
+            pluginInterface->logMessage(VmiCore::Plugin::LogLevel::error,
+                                        LOG_FILENAME,
+                                        "Error loading configuration: " + std::string(exc.what()));
+            return false;
+        }
+
+        try
+        {
+            pluginInterface->registerProcessStartEvent(processStartCallback);
+        }
+        catch (const std::exception&)
+        {
+            pluginInterface->logMessage(
+                VmiCore::Plugin::LogLevel::error, LOG_FILENAME, "Could not register ProcessStart Event");
+            return false;
+        }
+
+        try
+        {
+            pluginInterface->registerShutdownEvent(shutDownCallback);
+        }
+        catch (const std::exception&)
+        {
+            pluginInterface->logMessage(
+                VmiCore::Plugin::LogLevel::error, LOG_FILENAME, "Could not register ShutDown Event");
+            return false;
+        }
+
+        pluginInterface->logMessage(VmiCore::Plugin::LogLevel::info, LOG_FILENAME, "apiTracing Plugin: init end");
+
+        auto runningProcesses = pluginInterface->getRunningProcesses();
+        for (auto const& process : *runningProcesses)
+        {
+            pluginInterface->logMessage(VmiCore::Plugin::LogLevel::info, LOG_FILENAME, *process->fullName);
```

Comment:
I don't think it makes sense to log the process names here. They are already logged by the ActiveProcessSupervisor. At least make the log level `debug`.